release-please-config.json

```diff
@@ -0,0 +1,19 @@
+{
+  "packages": {
+    "solutions/landing-zone": {
+      "package-name": "landing-zone",
```

Comment:
I think this config would create a tag like `landing-zone-v0.0.1`, fetched with:

`kpt pkg get https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit.git/solutions/landing-zone@landing-zone-v0.0.1`

if we want to use the [kpt monorepo](https://kpt.dev/book/03-packages/08-publishing-a-package?id=monorepo-versioning) tagging convention we might have to put the full path for package-name (`solutions/landing-zone`) and add `"tag-separator": "/"` [configuration](https://github.com/google-github-actions/release-please-action#configuration)?

this should create a tag like `solutions/landing-zone/v0.0.1`, fetched with:

`kpt pkg get https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit.git/solutions/landing-zone@v0.0.1`

some pros and cons either way...

Reply:
Good point. I'll test that out in my fork to see how the `tag-separator` works. I just took a look there and it does indeed create a tag that matches `landing-zone-v0.0.1`  https://github.com/cartyc/pubsec-declarative-toolkit/tags .

Reply:
I tried the slash tag-separator on one of my private repos...seems to be working

Reply:
Awesome!